common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -335,6 +323,118 @@
                 });
 
             });
+    </script>
+}
+
+@bootSystemJs() = {
+
+    <script>
+        @Html(Static.js.es6ModuleLoader)
+        @Html(Static.js.systemJs)
+        @Html(Static.js.systemJsAppConfig)
+
+        @if(!play.Play.isDev()) {
+            @Html(Static.js.systemJsBundleConfig)
```

Comment:
I've been testing bundles in Prod today, have submitted fixes to issues regarding bundle arithmetic